dts/riscv/it8xxx2.dtsi

```diff
@@ -42,9 +43,174 @@
 				write-block-size = <1>;
 			};
 		};
-		pinmux: pinmux@f016f0 {
+		pinmuxa: pinmux@f01610 {
 			compatible = "ite,it8xxx2-pinmux";
-			reg = <0x00f016f0 0x0010>;
+			reg = <0x00f01610 0x0008>;
+			label = "PINMUXA";
+			func3_gcr =     <NO_FUNC  NO_FUNC  NO_FUNC  NO_FUNC
+					 0xf02032 0xf02032 0xf016f0 0xf016f0>;
+			func3_en_mask = <0        0        0        0
+					 0x02     0x02     0x10     0x0C    >;
+			func4_gcr =     <NO_FUNC  NO_FUNC  NO_FUNC  NO_FUNC
+					 NO_FUNC  NO_FUNC  NO_FUNC  NO_FUNC >;
+			func4_en_mask = <0        0        0        0
+					 0        0        0        0       >;
+		};
+		pinmuxb: pinmux@f01618 {
+			compatible = "ite,it8xxx2-pinmux";
+			reg = <0x00f01618 0x0008>;
+			label = "PINMUXB";
+			func3_gcr =     <0xf016f5 0xf016f5 NO_FUNC NO_FUNC
+					 NO_FUNC  NO_FUNC  NO_FUNC 0xf01600>;
+			func3_en_mask = <0x01     0x02     0       0
```

Comment:
Referencing IT81202 datasheet v0.3.2.1:
For GPIOB0,  and GPIOB1.  The U1CTRL field needs to be set to >0.  Where does this happen?

Reply:
U1CTRL>0 has been defined in soc.c(IT8XXX2_GPIO_GRC1 |= BIT(0)).

Reply:
Is this in a different pull request?  I don't see this in soc/riscv/riscv_ite/it82xxx2/soc.c.

Reply:
Never mind, I see it in this PR.